app/views/runs/show.slim

```diff
@@ -35,6 +35,7 @@
       - if @run.video_url.present?
         li = link_to 'proof', @run.video_url
       li = link_to 'stats', run_stats_path(@run)
+      li = link_to 'graphs', run_graphs_path(@run)
```

Comment:
In my local testing I didn't have the speedrun.com or proof link, but that does seem very crowded.  I'll see what I can change to reduce that.